src/test/java/org/takes/misc/VerboseIteratorTest.java

```diff
@@ -0,0 +1,103 @@
+package org.takes.misc;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableWithSize;
+import org.junit.Test;
+
+/**
+ * Tests for {@link VerboseIterator}.
+ * 
+ * @author marcus.sanchez (sanchez.marcus@gmail.com)
+ * @version $Id$
+ * @since 0.15.1
+ */
+public class VerboseIteratorTest {
+
+    /**
+     * VerboseIterator can return next value on a valid list
+     */
+    @Test
+    public void returnsNextValue() {
+        final List<String> VALID_LIST = Arrays.asList(
+                "Accept: text/plain",
+                "Accept-Charset: utf-8",
+                "Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==",
+                "Cache-Control: no-cache",
+                "From: user@example.com");
```

Comment:
@m4solutions ending parentheses should do on the line below. see article on indentation and brackets.